specs/ems/estimation-measurement-vocab-1-0.html

```diff
@@ -0,0 +1,2155 @@
+            </tr>
+            <tr>
+              <td>qudt-quantity:</td>
+              <td><a
+                  href="http://www.qudt.org/qudt/owl/1.0.0/quantity/index.html">&lt;http://data.nasa.gov/qudt/owl/quantity#&gt;</a>
+              </td>
+            </tr>
+            <tr>
+              <td>qudt-unit:</td>
+              <td><a
+                  href="http://www.qudt.org/qudt/owl/1.0.0/unit/index.html">&lt;http://data.nasa.gov/qudt/owl/unit#&gt;</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+      </section>
+
+    </section>
+
+    <section id="ems.3AFormat">
+
+      <h3 id="ems.3AFormat">ems:Format</h3>
+
+      <p>ems:Format is the class of all format URIs.</p>
+
+    </section>
+
+    <section id="ems_WbsFormat">
+
+      <h3 id="ems_WbsFormat">ems:WbsFormat</h3>
+
+      <p>ems:WbsFormat is the class of all WBS format URIs. This class is the range of the <a
+          href="https://archive.open-services.net/bin/view/Main/MetricsEmsWorkBreakdownStructure.html#ems_wbsFormat">ems:wbsFormat</a>
+        property
+        of an <a
+          href="https://archive.open-services.net/bin/view/Main/MetricsEmsWorkBreakdownStructure.html">ems:WorkBreakdownStructure</a>
+        resource. The following table contains the standard WBS format URIs:</p>
+
+      <h6 id="Table-of-ems.3AWbsFormat-Standard-URIs">Table of ems:WbsFormat Standard URIs</h6>
+
+      <table>
+        <thead>
+          <tr>
+            <th><strong>URI</strong></th>
+            <th><strong>Description</strong></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>&lt;http://schemas.microsoft.com/project/2007&gt;</td>
+            <td><a href="http://schemas.microsoft.com/project/2007/">Microsoft Office Project 2007 XML Data Interchange
+                Schema</a></td>
+          </tr>
+        </tbody>
+      </table>
+
+    </section>
+
+    <section id="ems_TimeMetric">
+
+      <h3 id="ems_TimeMetric">ems:TimeMetric</h3>
+
+      <p>ems:TimeMetric is the class of all time metrics. It is a subclass of ems:Metric. Time metrics quantify both
+        when events occur and how much time passes between two events. These two types of time metric are closely
+        related but different. Here we use the term <em>event</em> to mean some instant that marks a point in time, but
+        that itself occupies no duration. The length of time that passes between two project events is typically
+        measured in hours, days, weeks, etc.</p>
+
+      <p>The point in time at which an event occurs is measured with reference to some calendar that establishes an
+        origin point in time. For example, in the Gregorian calendar, points in time are measured as the amount of time
+        that has elapsed since a certain historical event that is denoted as year 1 AD. Points in <a
+          href="http://en.wikipedia.org/wiki/Unix_time">Unix time</a> as measured as the elapsed time since the
```

Comment:
@jamsden what is this? normative reference? non-normative reference? just a link?